spec/lib/phobos/producer_spec.rb

```diff
@@ -16,7 +16,7 @@ class TestProducer1
 
       expect(TestProducer1.producer)
         .to receive(:publish_list)
-        .with([{ topic: 'topic', payload: 'message', key: 'key' }])
+        .with([{ topic: 'topic', payload: 'message', key: 'key', partition_key: nil }])
 
       subject.producer.publish('topic', 'message', 'key')
```

Comment:
shouldn't this spec be updated use the new 4th argument?

Reply:
Nope, I still want to make sure that for backwards compatibility it still works correctly if no partition key is given.